packages/rpc-core/src/rpc-methods/__tests__/get-stake-minimum-delegation-test.ts

```diff
@@ -0,0 +1,26 @@
+import { createHttpTransport, createJsonRpc } from '@solana/rpc-transport';
+import type { Rpc } from '@solana/rpc-transport/dist/types/json-rpc-types';
+import fetchMock from 'jest-fetch-mock-fork';
+import { SolanaRpcMethods, createSolanaRpcApi } from '../../index';
+import { Commitment } from '../common';
+
+describe('getStakeMinimumDelegation', () => {
+    let rpc: Rpc<SolanaRpcMethods>;
+    beforeEach(() => {
+        fetchMock.resetMocks();
+        fetchMock.dontMock();
+        rpc = createJsonRpc<SolanaRpcMethods>({
+            api: createSolanaRpcApi(),
+            transport: createHttpTransport({ url: 'http://127.0.0.1:8899' }),
+        });
+    });
+    (['confirmed', 'finalized', 'processed'] as Commitment[]).forEach(commitment => {
+        describe(`when called with \`${commitment}\` commitment`, () => {
+            it('returns the result as a bigint wrapped in an RpcResponse', async () => {
+                expect.assertions(1);
+                const result = await rpc.getStakeMinimumDelegation({ commitment }).send();
+                expect(result.value).toEqual(expect.any(BigInt));
```

Comment:
Should we drop a `TODO` here to actually fetch the minimum delegation value from the stake program?

Probably not, now that I think about it. This test shouldn't really be in the business of _testing_ the underlying method. It should stop short of testing the type of the responses, as you've done here.

Reply:
Yeah exactly, that was my intention